content/docs/2.7/scalers/azure-data-explorer.md

```diff
@@ -0,0 +1,240 @@
++++
+title = "Azure Data Explorer"
+layout = "scaler"
+availability = "v2.7+"
+maintainer = "Community"
+description = "Scale applications based on Azure Data Explorer query result."
+go_file = "azure_data_explorer_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `azure-data-explorer` trigger that scales based on an Azure Data Explorer query result.
+
+```yaml
+triggers:
+- type: azure-data-explorer
+  metadata:
+    endpoint: "DATA_EXPLORER_ENDPOINT"
+    databaseName: "DATA_EXPLORER_DATABASE_NAME"
```

Comment:
Yes because it gives the impression that these are env variable names; maybe just use examples? It's ok to use `keda` as a resource name if need be.

We might want to update the LA docs as well then.

Reply:
done. 
I don't mind to do the same changes for Log Analytics, just LMK if you think it should be in different PR or part of this one.